style/git/README.md

```diff
@@ -83,6 +83,22 @@ The body of the commit message should use a style similar to the one proposed
 in this [article by tpope][tpope]. The body, just like the subject, should use
 an imperative tone.
 
+Another nice thing you can do in the body of your commit is state what the `issue` is that you're working on, and what your code changes and `fix` entailed.
```

Comment:
I'd love it if we somehow mentioned in there that it's the _why_ of this body that is so important. Commit messages that repeat _what_ they're doing (repeat because _what_ is in the diff itself) aren't terribly useful. Why they chose this solution is super interesting.